2020/02/01 22:33:07 [emerg] 4512#0: bind() to 0.0.0.0:8080 failed (48: Address already in use)
2020/02/01 22:33:07 [emerg] 4512#0: bind() to 0.0.0.0:80 failed (48: Address already in use)
2020/02/01 22:33:07 [emerg] 4512#0: bind() to 0.0.0.0:8080 failed (48: Address already in use)
2020/02/01 22:33:07 [emerg] 4512#0: bind() to 0.0.0.0:80 failed (48: Address already in use)
2020/02/01 22:33:07 [emerg] 4512#0: bind() to 0.0.0.0:8080 failed (48: Address already in use)
2020/02/01 22:33:07 [emerg] 4512#0: bind() to 0.0.0.0:80 failed (48: Address already in use)
2020/02/01 22:33:07 [emerg] 4512#0: bind() to 0.0.0.0:8080 failed (48: Address already in use)
2020/02/01 22:33:07 [emerg] 4512#0: bind() to 0.0.0.0:80 failed (48: Address already in use)
2020/02/01 22:33:34 [emerg] 4518#0: bind() to 0.0.0.0:8080 failed (48: Address already in use)
2020/02/01 22:33:34 [emerg] 4518#0: bind() to 0.0.0.0:80 failed (48: Address already in use)
2020/02/01 22:33:34 [emerg] 4518#0: bind() to 0.0.0.0:8080 failed (48: Address already in use)
2020/02/01 22:33:34 [emerg] 4518#0: bind() to 0.0.0.0:80 failed (48: Address already in use)
2020/02/01 22:33:34 [emerg] 4518#0: bind() to 0.0.0.0:8080 failed (48: Address already in use)
2020/02/01 22:33:34 [emerg] 4518#0: bind() to 0.0.0.0:80 failed (48: Address already in use)
2020/02/01 23:15:58 [error] 4523#0: *4 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/01 23:16:00 [error] 4523#0: *4 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/01 23:16:39 [error] 4523#0: *4 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/01 23:16:39 [error] 4523#0: *4 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/01 23:22:09 [emerg] 5128#0: bind() to 0.0.0.0:80 failed (48: Address already in use)
2020/02/01 23:22:09 [emerg] 5128#0: bind() to 0.0.0.0:80 failed (48: Address already in use)
2020/02/01 23:22:09 [emerg] 5128#0: bind() to 0.0.0.0:80 failed (48: Address already in use)
2020/02/01 23:22:09 [emerg] 5128#0: bind() to 0.0.0.0:80 failed (48: Address already in use)
2020/02/02 08:49:34 [emerg] 955#0: bind() to 0.0.0.0:80 failed (48: Address already in use)
2020/02/02 08:49:34 [emerg] 955#0: bind() to 0.0.0.0:80 failed (48: Address already in use)
2020/02/02 08:49:34 [emerg] 955#0: bind() to 0.0.0.0:80 failed (48: Address already in use)
2020/02/02 08:49:34 [emerg] 955#0: bind() to 0.0.0.0:80 failed (48: Address already in use)
2020/02/03 10:01:52 [emerg] 11397#0: bind() to 0.0.0.0:80 failed (48: Address already in use)
2020/02/03 10:01:52 [emerg] 11397#0: bind() to 0.0.0.0:80 failed (48: Address already in use)
2020/02/03 10:01:52 [emerg] 11397#0: bind() to 0.0.0.0:80 failed (48: Address already in use)
2020/02/03 10:01:52 [emerg] 11397#0: bind() to 0.0.0.0:80 failed (48: Address already in use)
2020/02/03 10:07:35 [emerg] 11470#0: bind() to 0.0.0.0:8080 failed (48: Address already in use)
2020/02/03 10:07:35 [emerg] 11470#0: bind() to 0.0.0.0:80 failed (48: Address already in use)
2020/02/03 10:07:35 [emerg] 11470#0: bind() to 0.0.0.0:8080 failed (48: Address already in use)
2020/02/03 10:07:35 [emerg] 11470#0: bind() to 0.0.0.0:80 failed (48: Address already in use)
2020/02/03 10:07:35 [emerg] 11470#0: bind() to 0.0.0.0:8080 failed (48: Address already in use)
2020/02/03 10:07:35 [emerg] 11470#0: bind() to 0.0.0.0:80 failed (48: Address already in use)
2020/02/03 10:07:35 [emerg] 11470#0: bind() to 0.0.0.0:8080 failed (48: Address already in use)
2020/02/03 10:07:35 [emerg] 11470#0: bind() to 0.0.0.0:80 failed (48: Address already in use)
2020/02/03 10:07:35 [emerg] 11470#0: bind() to 0.0.0.0:8080 failed (48: Address already in use)
2020/02/03 10:07:35 [emerg] 11470#0: bind() to 0.0.0.0:80 failed (48: Address already in use)
2020/02/03 10:07:35 [emerg] 11470#0: still could not bind()
2020/02/22 23:38:51 [error] 9003#0: *523 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/22 23:39:18 [error] 9003#0: *523 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /topics/6 HTTP/1.1", upstream: "http://127.0.0.1:10080/topics/6", host: "larabbs.com", referrer: "http://larabbs.com/topics/6/edit"
2020/02/22 23:39:18 [error] 9003#0: *523 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/topics/6"
2020/02/22 23:39:20 [error] 9003#0: *523 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/topics/6/edit"
2020/02/24 21:13:24 [error] 9003#0: *1345 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/24 21:13:27 [error] 9003#0: *1345 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/24 21:13:27 [error] 9003#0: *1345 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/24 21:13:28 [error] 9003#0: *1345 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/24 21:13:28 [error] 9003#0: *1345 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/24 21:20:54 [error] 9003#0: *1351 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/24 21:20:55 [error] 9003#0: *1351 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/24 21:20:55 [error] 9003#0: *1351 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/24 21:21:04 [error] 9003#0: *1351 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/24 21:21:04 [error] 9003#0: *1351 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/24 21:21:05 [error] 9003#0: *1351 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/24 21:21:05 [error] 9003#0: *1351 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:12:12 [error] 9003#0: *1359 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:12:13 [error] 9003#0: *1359 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:12:14 [error] 9003#0: *1359 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:12:14 [error] 9003#0: *1359 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:12:14 [error] 9003#0: *1359 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:12:14 [error] 9003#0: *1359 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:12:15 [error] 9003#0: *1359 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:12:15 [error] 9003#0: *1359 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:12:15 [error] 9003#0: *1359 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:12:15 [error] 9003#0: *1359 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:12:15 [error] 9003#0: *1359 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:12:15 [error] 9003#0: *1359 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:12:15 [error] 9003#0: *1359 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:12:15 [error] 9003#0: *1359 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:12:15 [error] 9003#0: *1359 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:12:15 [error] 9003#0: *1359 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:12:15 [error] 9003#0: *1359 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:12:16 [error] 9003#0: *1359 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:12:16 [error] 9003#0: *1359 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:12:16 [error] 9003#0: *1359 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:12:16 [error] 9003#0: *1359 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:12:16 [error] 9003#0: *1359 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:12:16 [error] 9003#0: *1359 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:12:16 [error] 9003#0: *1359 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:12:16 [error] 9003#0: *1359 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:12:16 [error] 9003#0: *1359 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:12:16 [error] 9003#0: *1359 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:12:17 [error] 9003#0: *1359 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:12:17 [error] 9003#0: *1359 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:12:17 [error] 9003#0: *1359 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:12:17 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:12:17 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:12:17 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:12:17 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:12:17 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:12:17 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:12:17 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:12:17 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:12:18 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:12:18 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:12:18 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:12:18 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:13:07 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:13:07 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:13:09 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:13:09 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:13:09 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:13:09 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:13:10 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:13:10 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:13:10 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:13:10 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:13:10 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:13:10 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:13:11 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:13:11 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:13:11 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:13:11 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:13:11 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:13:11 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:13:11 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:13:11 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:13:11 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:13:12 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:13:12 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:13:12 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:14:01 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:14:01 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:14:01 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:14:02 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:14:02 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:14:02 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:14:03 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:14:03 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:14:04 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:14:04 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:14:04 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:14:04 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:14:04 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:14:05 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:14:58 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:14:58 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:14:59 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:14:59 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:14:59 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:14:59 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:14:59 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:14:59 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:15:30 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:15:30 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:15:31 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:15:31 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:15:31 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:15:31 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:15:32 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:15:32 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:15:32 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:15:32 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:15:32 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 00:15:32 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:15:32 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:15:32 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:15:33 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:15:33 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:15:33 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 00:15:33 [error] 9003#0: *1390 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 12:26:06 [error] 9003#0: *1487 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 12:26:07 [error] 9003#0: *1487 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 12:26:09 [error] 9003#0: *1487 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 12:51:56 [error] 9003#0: *1514 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /topics/98 HTTP/1.1", upstream: "http://127.0.0.1:10080/topics/98", host: "larabbs.com", referrer: "http://larabbs.com/categories/2"
2020/02/25 13:00:34 [error] 9003#0: *1523 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 13:01:05 [error] 9003#0: *1525 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 13:01:06 [error] 9003#0: *1525 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 13:01:23 [error] 9003#0: *1532 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 13:01:24 [error] 9003#0: *1532 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 21:23:00 [error] 9003#0: *1549 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 21:23:00 [error] 9003#0: *1550 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /css/app.css?id=e52e6b7b0c8246695365 HTTP/1.1", upstream: "http://127.0.0.1:10080/css/app.css?id=e52e6b7b0c8246695365", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/02/25 21:23:01 [error] 9003#0: *1550 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/02/25 21:23:32 [error] 9003#0: *1550 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/03/05 17:49:40 [error] 44805#0: *3784 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /users/10 HTTP/1.1", upstream: "http://127.0.0.1:10080/users/10", host: "larabbs.com", referrer: "http://larabbs.com/categories/2"
2020/03/05 17:49:41 [error] 44805#0: *3784 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/users/10"
2020/03/07 09:45:12 [emerg] 44820#0: unknown directive " " in /Users/yanyiming/nginx/servers/movies.conf:2
2020/03/07 10:11:38 [emerg] 44820#0: unknown directive " " in /Users/yanyiming/nginx/servers/movies.conf:2
2020/03/07 15:30:19 [emerg] 44820#0: unknown directive " " in /Users/yanyiming/nginx/servers/movies.conf:2
2020/03/07 15:30:32 [emerg] 44820#0: unknown directive " " in /Users/yanyiming/nginx/servers/movies.conf:2
2020/03/07 15:30:40 [emerg] 44820#0: unknown directive " " in /Users/yanyiming/nginx/servers/movies.conf:2
2020/03/07 15:30:59 [emerg] 44820#0: unknown directive " " in /Users/yanyiming/nginx/servers/movies.conf:3
2020/03/07 15:31:00 [emerg] 44820#0: unknown directive " " in /Users/yanyiming/nginx/servers/movies.conf:3
2020/03/07 15:40:58 [error] 50773#0: *1 rewrite or internal redirection cycle while internally redirecting to "/index.php", client: 127.0.0.1, server: movies.com, request: "GET /favicon.ico HTTP/1.1", host: "movies.com:81", referrer: "http://movies.com:81/"
2020/03/07 16:17:03 [error] 50773#0: *39 rewrite or internal redirection cycle while internally redirecting to "/index.php", client: 127.0.0.1, server: movies.com, request: "GET /favicon.ico HTTP/1.1", host: "movies.com:81", referrer: "http://movies.com:81/"
2020/03/17 23:10:39 [emerg] 85760#0: bind() to 0.0.0.0:8080 failed (48: Address already in use)
2020/03/17 23:10:39 [emerg] 85760#0: bind() to 0.0.0.0:81 failed (48: Address already in use)
2020/03/17 23:10:39 [emerg] 85760#0: bind() to 0.0.0.0:80 failed (48: Address already in use)
2020/03/17 23:10:39 [emerg] 85760#0: bind() to 0.0.0.0:8080 failed (48: Address already in use)
2020/03/17 23:10:39 [emerg] 85760#0: bind() to 0.0.0.0:81 failed (48: Address already in use)
2020/03/17 23:10:39 [emerg] 85760#0: bind() to 0.0.0.0:80 failed (48: Address already in use)
2020/03/17 23:10:39 [emerg] 85760#0: bind() to 0.0.0.0:8080 failed (48: Address already in use)
2020/03/17 23:10:39 [emerg] 85760#0: bind() to 0.0.0.0:81 failed (48: Address already in use)
2020/03/17 23:10:39 [emerg] 85760#0: bind() to 0.0.0.0:80 failed (48: Address already in use)
2020/03/18 00:19:07 [error] 50773#0: *440 open() "/usr/local/Cellar/nginx/1.17.8/html/version" failed (2: No such file or directory), client: 127.0.0.1, server: localhost, request: "GET /version?timeout=32s HTTP/1.1", host: "localhost:8080"
2020/03/18 00:19:07 [error] 50773#0: *441 open() "/usr/local/Cellar/nginx/1.17.8/html/api" failed (2: No such file or directory), client: 127.0.0.1, server: localhost, request: "GET /api?timeout=32s HTTP/1.1", host: "localhost:8080"
2020/03/18 00:19:07 [error] 50773#0: *441 open() "/usr/local/Cellar/nginx/1.17.8/html/apis" failed (2: No such file or directory), client: 127.0.0.1, server: localhost, request: "GET /apis?timeout=32s HTTP/1.1", host: "localhost:8080"
2020/03/18 00:19:07 [error] 50773#0: *441 open() "/usr/local/Cellar/nginx/1.17.8/html/api" failed (2: No such file or directory), client: 127.0.0.1, server: localhost, request: "GET /api?timeout=32s HTTP/1.1", host: "localhost:8080"
2020/03/18 00:19:07 [error] 50773#0: *441 open() "/usr/local/Cellar/nginx/1.17.8/html/apis" failed (2: No such file or directory), client: 127.0.0.1, server: localhost, request: "GET /apis?timeout=32s HTTP/1.1", host: "localhost:8080"
2020/03/18 00:19:07 [error] 50773#0: *441 open() "/usr/local/Cellar/nginx/1.17.8/html/api" failed (2: No such file or directory), client: 127.0.0.1, server: localhost, request: "GET /api?timeout=32s HTTP/1.1", host: "localhost:8080"
2020/03/18 00:19:07 [error] 50773#0: *441 open() "/usr/local/Cellar/nginx/1.17.8/html/apis" failed (2: No such file or directory), client: 127.0.0.1, server: localhost, request: "GET /apis?timeout=32s HTTP/1.1", host: "localhost:8080"
2020/03/18 00:19:07 [error] 50773#0: *442 open() "/usr/local/Cellar/nginx/1.17.8/html/api" failed (2: No such file or directory), client: 127.0.0.1, server: localhost, request: "GET /api?timeout=32s HTTP/1.1", host: "localhost:8080"
2020/03/18 00:19:07 [error] 50773#0: *442 open() "/usr/local/Cellar/nginx/1.17.8/html/apis" failed (2: No such file or directory), client: 127.0.0.1, server: localhost, request: "GET /apis?timeout=32s HTTP/1.1", host: "localhost:8080"
2020/03/18 00:19:07 [error] 50773#0: *442 open() "/usr/local/Cellar/nginx/1.17.8/html/api" failed (2: No such file or directory), client: 127.0.0.1, server: localhost, request: "GET /api?timeout=32s HTTP/1.1", host: "localhost:8080"
2020/03/18 00:19:07 [error] 50773#0: *442 open() "/usr/local/Cellar/nginx/1.17.8/html/apis" failed (2: No such file or directory), client: 127.0.0.1, server: localhost, request: "GET /apis?timeout=32s HTTP/1.1", host: "localhost:8080"
2020/03/18 00:19:07 [error] 50773#0: *442 open() "/usr/local/Cellar/nginx/1.17.8/html/api" failed (2: No such file or directory), client: 127.0.0.1, server: localhost, request: "GET /api?timeout=32s HTTP/1.1", host: "localhost:8080"
2020/03/18 00:19:07 [error] 50773#0: *442 open() "/usr/local/Cellar/nginx/1.17.8/html/apis" failed (2: No such file or directory), client: 127.0.0.1, server: localhost, request: "GET /apis?timeout=32s HTTP/1.1", host: "localhost:8080"
2020/03/18 07:12:53 [error] 50773#0: *443 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/03/18 07:21:21 [error] 50773#0: *445 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/03/18 07:21:22 [error] 50773#0: *445 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: larabbs.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:10080/favicon.ico", host: "larabbs.com", referrer: "http://larabbs.com/"
2020/03/18 07:22:26 [error] 50773#0: *445 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: larabbs.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:10080/", host: "larabbs.com"
2020/03/29 12:01:18 [error] 7488#0: *1 open() "/usr/local/Cellar/nginx/1.17.8/html/stream/1.m3u8" failed (2: No such file or directory), client: 127.0.0.1, server: localhost, request: "GET /stream/1.m3u8 HTTP/1.1", host: "127.0.0.1:8080"
2020/03/29 12:01:18 [error] 7488#0: *2 open() "/usr/local/Cellar/nginx/1.17.8/html/stream/1.m3u8" failed (2: No such file or directory), client: 127.0.0.1, server: localhost, request: "GET /stream/1.m3u8 HTTP/1.0", host: "127.0.0.1:8080"
2020/03/29 12:02:33 [error] 7488#0: *3 open() "/usr/local/Cellar/nginx/1.17.8/html/stream/1.m3u8" failed (2: No such file or directory), client: 127.0.0.1, server: localhost, request: "GET /stream/1.m3u8 HTTP/1.1", host: "127.0.0.1:8080"
2020/03/29 12:02:33 [error] 7488#0: *3 open() "/usr/local/Cellar/nginx/1.17.8/html/robots.txt" failed (2: No such file or directory), client: 127.0.0.1, server: localhost, request: "GET /robots.txt HTTP/1.1", host: "127.0.0.1:8080"
2020/03/29 12:02:35 [error] 7488#0: *3 open() "/usr/local/Cellar/nginx/1.17.8/html/favicon.ico" failed (2: No such file or directory), client: 127.0.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", host: "127.0.0.1:8080", referrer: "http://127.0.0.1:8080/stream/1.m3u8"
2020/03/29 12:02:55 [error] 7488#0: *3 "/usr/local/Cellar/nginx/1.17.8/html/stream/index.html" is not found (2: No such file or directory), client: 127.0.0.1, server: localhost, request: "GET /stream/ HTTP/1.1", host: "127.0.0.1:8080"
2020/03/29 12:02:57 [error] 7488#0: *3 "/usr/local/Cellar/nginx/1.17.8/html/stream/index.html" is not found (2: No such file or directory), client: 127.0.0.1, server: localhost, request: "GET /stream/ HTTP/1.1", host: "127.0.0.1:8080"
2020/03/29 12:02:58 [error] 7488#0: *3 "/usr/local/Cellar/nginx/1.17.8/html/stream/index.html" is not found (2: No such file or directory), client: 127.0.0.1, server: localhost, request: "GET /stream/ HTTP/1.1", host: "127.0.0.1:8080"
2020/03/29 12:02:58 [error] 7488#0: *3 "/usr/local/Cellar/nginx/1.17.8/html/stream/index.html" is not found (2: No such file or directory), client: 127.0.0.1, server: localhost, request: "GET /stream/ HTTP/1.1", host: "127.0.0.1:8080"
2020/03/29 12:02:59 [error] 7488#0: *3 "/usr/local/Cellar/nginx/1.17.8/html/stream/index.html" is not found (2: No such file or directory), client: 127.0.0.1, server: localhost, request: "GET /stream/ HTTP/1.1", host: "127.0.0.1:8080"
2020/03/29 12:02:59 [error] 7488#0: *3 "/usr/local/Cellar/nginx/1.17.8/html/stream/index.html" is not found (2: No such file or directory), client: 127.0.0.1, server: localhost, request: "GET /stream/ HTTP/1.1", host: "127.0.0.1:8080"
2020/03/29 12:08:06 [error] 7488#0: *15 open() "/usr/local/Cellar/nginx/1.17.8/html/live/test" failed (2: No such file or directory), client: 127.0.0.1, server: localhost, request: "GET /live/test HTTP/1.1", host: "127.0.0.1:8080"
2020/03/29 12:08:06 [error] 7488#0: *16 open() "/usr/local/Cellar/nginx/1.17.8/html/live/test" failed (2: No such file or directory), client: 127.0.0.1, server: localhost, request: "GET /live/test HTTP/1.0", host: "127.0.0.1:8080"
2020/03/29 12:08:16 [error] 7488#0: *17 open() "/usr/local/Cellar/nginx/1.17.8/html/live/test.m3u8" failed (2: No such file or directory), client: 127.0.0.1, server: localhost, request: "GET /live/test.m3u8 HTTP/1.1", host: "127.0.0.1:8080"
2020/03/29 12:08:16 [error] 7488#0: *18 open() "/usr/local/Cellar/nginx/1.17.8/html/live/test.m3u8" failed (2: No such file or directory), client: 127.0.0.1, server: localhost, request: "GET /live/test.m3u8 HTTP/1.0", host: "127.0.0.1:8080"
2020/03/29 12:08:21 [error] 7488#0: *20 open() "/usr/local/Cellar/nginx/1.17.8/html/live/test.m3u8" failed (2: No such file or directory), client: 127.0.0.1, server: localhost, request: "GET /live/test.m3u8 HTTP/1.1", host: "127.0.0.1:8080"
2020/03/29 12:08:21 [error] 7488#0: *21 open() "/usr/local/Cellar/nginx/1.17.8/html/live/test.m3u8" failed (2: No such file or directory), client: 127.0.0.1, server: localhost, request: "GET /live/test.m3u8 HTTP/1.0", host: "127.0.0.1:8080"
2020/03/29 12:08:33 [error] 7488#0: *23 open() "/usr/local/Cellar/nginx/1.17.8/html/live/test" failed (2: No such file or directory), client: 127.0.0.1, server: localhost, request: "GET /live/test HTTP/1.1", host: "127.0.0.1:8080"
2020/03/29 12:08:33 [error] 7488#0: *24 open() "/usr/local/Cellar/nginx/1.17.8/html/live/test" failed (2: No such file or directory), client: 127.0.0.1, server: localhost, request: "GET /live/test HTTP/1.0", host: "127.0.0.1:8080"
2020/03/29 12:08:36 [error] 7488#0: *25 open() "/usr/local/Cellar/nginx/1.17.8/html/live/test.m3u8" failed (2: No such file or directory), client: 127.0.0.1, server: localhost, request: "GET /live/test.m3u8 HTTP/1.1", host: "127.0.0.1:8080"
2020/03/29 12:08:36 [error] 7488#0: *26 open() "/usr/local/Cellar/nginx/1.17.8/html/live/test.m3u8" failed (2: No such file or directory), client: 127.0.0.1, server: localhost, request: "GET /live/test.m3u8 HTTP/1.0", host: "127.0.0.1:8080"
2020/04/03 07:59:26 [error] 37394#0: *85 rewrite or internal redirection cycle while internally redirecting to "/index.php", client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost:443"
2020/04/03 07:59:42 [error] 37394#0: *86 rewrite or internal redirection cycle while internally redirecting to "/index.php", client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "test.local:443"
2020/04/03 09:01:10 [error] 37581#0: *137 open() "/Users/yanyiming/wss_to_ws/robots.txt" failed (2: No such file or directory), client: 127.0.0.1, server: test.local, request: "GET /robots.txt HTTP/1.1", host: "test.local:443"
2020/04/03 09:01:39 [error] 37581#0: *137 open() "/Users/yanyiming/wss_to_ws/favicon.ico" failed (2: No such file or directory), client: 127.0.0.1, server: test.local, request: "GET /favicon.ico HTTP/1.1", host: "test.local:443", referrer: "http://test.local:443/"
2020/04/03 09:08:44 [emerg] 7487#0: no "ssl_certificate" is defined for the "listen ... ssl" directive in /Users/yanyiming/nginx/servers/wss_to_ws.conf:1
2020/04/03 09:11:04 [emerg] 7487#0: no "ssl_certificate" is defined for the "listen ... ssl" directive in /Users/yanyiming/nginx/servers/wss_to_ws.conf:1
2020/04/04 10:48:10 [emerg] 7487#0: no "ssl_certificate" is defined for the "listen ... ssl" directive in /Users/yanyiming/nginx/servers/wss_to_ws.conf:1
2020/04/10 17:46:45 [emerg] 7487#0: no "ssl_certificate" is defined for the "listen ... ssl" directive in /Users/yanyiming/nginx/servers/wss_to_ws.conf:1
2020/04/10 17:52:17 [emerg] 7487#0: no "ssl_certificate" is defined for the "listen ... ssl" directive in /Users/yanyiming/nginx/servers/wss_to_ws.conf:1
2020/04/10 18:12:04 [emerg] 7487#0: no "ssl_certificate" is defined for the "listen ... ssl" directive in /Users/yanyiming/nginx/servers/wss_to_ws.conf:1
2020/04/10 18:13:33 [emerg] 7487#0: no "ssl_certificate" is defined for the "listen ... ssl" directive in /Users/yanyiming/nginx/servers/wss_to_ws.conf:1
2020/04/10 18:19:10 [emerg] 7487#0: no "ssl_certificate" is defined for the "listen ... ssl" directive in /Users/yanyiming/nginx/servers/wss_to_ws.conf:1
2020/04/10 18:24:35 [emerg] 82355#0: bind() to 0.0.0.0:9503 failed (48: Address already in use)
2020/04/10 18:24:35 [emerg] 82355#0: bind() to 0.0.0.0:9503 failed (48: Address already in use)
2020/04/10 18:24:35 [emerg] 82355#0: bind() to 0.0.0.0:9503 failed (48: Address already in use)
2020/04/10 18:24:35 [emerg] 82355#0: bind() to 0.0.0.0:9503 failed (48: Address already in use)
2020/04/10 19:00:09 [error] 83371#0: *1 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: ws.localhost, request: "GET /broadcast HTTP/1.1", upstream: "http://127.0.0.1:5200/broadcast", host: "ws.localhost"
2020/04/10 19:02:08 [error] 83371#0: *3 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: ws.localhost, request: "GET /broadcast HTTP/1.1", upstream: "http://127.0.0.1:5200/broadcast", host: "ws.localhost"
2020/04/10 19:02:15 [error] 83371#0: *3 kevent() reported that connect() failed (61: Connection refused) while connecting to upstream, client: 127.0.0.1, server: ws.localhost, request: "GET /broadcast HTTP/1.1", upstream: "http://127.0.0.1:5200/broadcast", host: "ws.localhost"
